api-reference/10 UI Widgets/dxFileManager/1 Configuration/currentPathKeys.md

```diff
@@ -5,7 +5,22 @@ default: []
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies an array of path keys to the current location.
+
+---
+
+Each path part has each own key. For example, path "folder1/folder2" has two parts: 'folder1' with the 'f1' key and folder2 with the 'f2' key. To open this location, assign the ["f1","f2"] array of strings to the **currentPathKeys** option value. 
```

Comment:
Maybe it's just me, but I can't understand how should I know the path keys.

Reply:
Users know

Reply:
Users know.